packages/manager/apps/dedicated/client/app/ip/components/list/firewall/game/rule/delete/ip-ip-firewall-game-rule-delete.controller.js

```diff
@@ -5,16 +5,27 @@ export default /* @ngInject */ function RemoveGameFirewallRuleCtrl(
   $rootScope,
   Alerter,
   IpGameFirewall,
+  atInternet,
 ) {
   const self = this;
   const alert = 'ip_game_firewall_alert';
 
   self.datas = $scope.currentActionData;
   self.loading = false;
 
+  atInternet.trackClick({
+    name: $scope.data?.tracking,
+    type: 'action',
+  });
+
```

Comment:
I think you should test if $scope.data?.tracking is not undefined before calling atInternet.trackClick with it.
Unless we are sure data will allways be defined and has tracking also defined.